e2etest/bgp_test.go

```diff
@@ -152,8 +135,13 @@ var _ = ginkgo.Describe("BGP", func() {
 					},
 				},
 			},
-			Peers: peers,
+			Peers: peersForContainers(frrContainers, ipFamily),
+		}
+		for _, c := range frrContainers {
+			c.RouterConfig.IPFamily = ipFamily
+			c.NeighborConfig.IPFamily = ipFamily
 		}
+
 		for _, c := range frrContainers {
 			pairExternalFRRWithNodes(cs, c)
```

Comment:
I think we should have this in the loop above like you did here: https://github.com/metallb/metallb/blob/d0622145349a3c6687409722178e66e8f2f0e822/e2etest/bgp_test.go#L452)

Reply:
Yes did not notice while rebasing